vignettes/including-static-dashboards.Rmd

```diff
@@ -0,0 +1,103 @@
+---
+title: "Including Static Dashboards in Crunch"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Vignette Title}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+Crunch allows you to host Shiny apps on the Crunch Jupyter server. Shiny apps are great, but they can be difficult to program, and can be very slow if not designed properly. [Flexdashboards](https://rmarkdown.rstudio.com/flexdashboard/) provide an easy-to-use alternative to Shiny apps. Instead of re-rendering dynamically based on the user action action, Flexdashboards are effectively static websites which only include data which was live at the time of rendering. This can be desirable both because they are much easier to write, and because you can test and validate the dashboard's behavior before it goes to the client. Additionally, since you have finer control over when a dashboard is rendered, you can include more computationally expensive operations like fitting a machine learning model, or drawing a complicated graphic. 
+
+You can host Flexdashboards on Crunch's Shiny infrastructure using Shiny's `includeHTML` function. What `includeHTML` does is add an HTML file to the shiny UI. If all you include in the shiny UI is the Flexdashboard html file, then the Shiny app effectively becomes a static site. 
+
+### Basic workflow
+Follow these steps to include a Flexdashboard in your Shiny app: 
+1) First create a new Rmarkdown document with a Flexdashboard output option. 
+1) Render the Rmd file into an html file
+1) Uploaded the rendered html file to your app's Jupyter folder
+1) In your shiny UI code add `includeHTML(<filename>)` to the `ui` object. Your `app.R` file might look something like this:
+
+```
+library(shiny)
```

Comment:
`crunchy` loads `shiny`, so it's not required to `library()` both. Just `library(crunchy)`